packages/pluggableWidgets/datagrid-web/src/ui/Datagrid.scss

```diff
@@ -179,6 +184,30 @@ $dragging-color-effect: rgba(10, 19, 37, 0.8);
     & *:focus {
         outline: 0;
     }
+
+    .align-column-left {
+        justify-content: flex-start;
+
+        span {
```

Comment:
What is the use of the `span` elements inside column cells?

Reply:
Before was just the text, but now as we need to align the content and the outside element needs to grow (due to the sorting symbol in the right side of the column) i need to either introduce span (which fits more with text) or div :)